### DISCUSSION TITLE
Meeting notes - April 16th, 2024

### DISCUSSION THREAD

**Discussion by Chris-ECE** [UNECE/GSBPM_GAMSO_Revision#29] [General]
### Design of Dissemination
*	Regarding the comment from Saudi Arabia, there seems to be a lot of different items mentioned, and it’s not clear whether to include all of them, or if they can be encapsulated together, but no proposal is made if the latter. Identifying information security procedures has been discussed elsewhere, and Identifying the quality metrics would probably be an overarching process. The product promotion plan might be a communication activity (GAMSO) rather than a production activity.
*	Regarding the first comment, it was pointed out that the GSBPM “Task” task team developed a [third level of “tasks”](https://statswiki.unece.org/pages/viewpage.action?spaceKey=Standards&title=GSBPM+Tasks) (to differentiate GSBPM from GAMSO), following practice by some countries to develop their own list of tasks beneath GSBPM subprocesses. This team proposed the following dissemination-related tasks under 2.1:-
     -	_2.1.2 Design statistical outputs, products and services and necessary components_ (which could include the dissemination calendar)
     -	_2.1.3 Design dissemination procedure_ 
     -	_2.1.4 Design systems and tools for dissemination_
*	While those tasks refer to the design of dissemination procedures, systems and tools, the description of subprocess 2.1 does not include an explicit reference to them. Therefore, the description of subprocess 2.1 was modified to include the procedures used in the Disseminate phase (in addition to the systems and tools already mentioned there).

### Design of Coding
*	Coding in this context is interpreted to mean coding of data (and not writing code).
*	Antonio suggested this should belong to subprocess 2.5 (Design processing and analysis), although the description of 2.5 designs statistical processing methodology and doesn’t explicitly refer to coding.
*	The list of tasks developed by the GSBPM “Task” task team includes one on design of data coding, under 2.5:-
     -	_2.5.2 Design specifications for data coding_
*	While coding is not explicitly mentioned in the description of subprocess 2.5, it should be pointed out that many separate tasks have been identified under that subprocess, and referring to one of them in the description would motivate mentioning all of them. The same argument could be made for not separating design of coding into a separate subprocess, since this would motivate creating separate subprocesses for all the other tasks too.
*	The conclusion is that coding is already included within subprocess 2.5, even if it is not separately referred to.

<ins>Aside</ins>
* Antonio suggested an extra task dedicated to the (design of) calculation of precision/variance. This is not already covered by sample design, as it depends on response rates, nor in validate weights. Chris wasn’t sure whether the calculation of precision could be a quality indicator in the overarching processes. Edgardo suggested that to make an exhaustive list of tasks might be too much. It could be included in the evaluation phase.

### Design of Evaluation

Juan suggested that the design of evaluation is related both to:-
*	Production of the information needs defined in the Specify Needs phase; and
*	How well the process itself was performed, including lessons learned.

Antonio pointed out that the description of the Evaluate phase mentions that it may rely on inputs gathered throughout the different phases, and suggested that designing the evaluation is a way to be sure that the desired metrics for the evaluation have all been produced, and that this would belong in the Design phase somewhere.

Edgardo pointed out that the task 2.3.4 Design system for data collection management and monitoring, might produce useful metrics, but is narrowly focused on data collection. There doesn’t seem to be a task for monitoring more generally, but Edgardo suggested including one under 2.6 because it considered the broader workflow, from collection to dissemination, which the group decided to expand to “collection to evaluation” in the description of subprocess 2.6

Given that the “Task” task team is no longer in existence, it’s not clear what the process might be for revising the list of tasks. It was suggested that it could be done within this GSBPMN revision, for the sake of simplicity: To this end, a comment was added to the doc, suggesting to consider an extra 3rd-level task under 2.6 on Designing evaluation (or designing workflow from collection to evaluation).
It was also proposed to include the tasks in an annex to the GSBPM document, as a mechanism for ensuring they are kept updated.

However, Edgardo said that design of evaluation (and design of security), could be overarching in the sense of quality indicators, and suggested to put in a comment about making a link between 2.6 and quality indicators in overarching processes.

**Action:** To discuss further in the next call.

**Comment by vaccaricarlo** [comment]
Also the task "2.1.1 Design statistical concepts"  is dissemination-related
